### ISSUE TITLE
Here Geocoder

### ISSUE THREAD

**Issue by walling** [nchaulet/node-geocoder#118]
Is this module going to support [Here Geocoder](https://developer.here.com/api-explorer/rest/geocoder) [(documentation)](https://developer.here.com/rest-apis/documentation/geocoder/topics/resource-geocode.html)? It could be nice!

**Comment by nchaulet** [comment]
:+1:  could be a good provider to add

**Comment by walling** [comment]
I would like to contribute and send a pull request for the Here Geocoder including proper test coverage and everything. My employer really needs this, and asks if it can be merged and published the same day, when I submit it, assuming the code looks nice and all tests pass. @nchaulet, is it okay with you?

**Comment by nchaulet** [comment]
I invite you to contribute, If the code is ok it's quick to merge and bump a new version.

**Comment by walling** [comment]
Ok, thanks, I will look into it. My plan is to submit it on Monday.

**Comment by walling** [comment]
I looked into it already and send you a pull request. Please have a look, when you have time.

**Comment by nchaulet** [comment]
Just published node-geocoder v3.0.0 with the here geocoder

 the changelog  
```
remove command line geocoder: now in node-geocoder-cli
Remove getGeocoder from index module
Add Here geocoder
```

Thanks for your contribution :)
